README.md

```diff
@@ -20,6 +20,7 @@ The official OCS samples are divided in multiple categories depending on the sce
 | **<a href="basic_samples/SDS_TimeSeries">Types, Streams, and retreiving Data -- Time-Series</a>** <img src="./miscellaneous/images/app-type-getting-started.png" alt="getting-started icon"> | Covers some typical operations against the SDS, including client credential authentication, creating, and deleting types and streams.  This sample is based on Time-Series data.  Some tasks are different from the other "Types, Streams, and retreiving Data" task, so look at the task readme for details.  This is a recommended starting example, and a good a base for all other Tasks.  <a href="basic_samples/SDS_TimeSeries">Details</a>| <a href="basic_samples/SDS_TimeSeries/Python">Python</a> <br /> <a href="basic_samples/SDS_TimeSeries/DotNet/Try">DotNet</a> | [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OCS/SDS_TS_Python?branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=927&branchName=master) <br /> [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OCS/SDS_TS_DotNet?branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=926&branchName=master) |
 | **<a href="advanced_samples/UomsSample/Dotnet/UomsSample/UomsSample">UOM</a>** <img src="./miscellaneous/images/app-type-getting-started.png" alt="getting-started icon"> | Covers the basic functionality of the UOM system on OCS | <a href="advanced_samples/UomsSample/Dotnet/UomsSample/UomsSample">.NET</a> | [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OCS/UOM_DotNet?branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=928&branchName=master) |
+| **<a href="advanced_samples/CSV/dotnet/CSVtoOCS_pkce">CSV upload</a>**  | Shows sending a .csv file to OCS | <a href="advanced_samples/CSV/dotnet/CSVtoOCS_pkce">.NET</a> | [![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OCS/CSVtoOCS_DotNet?branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=1393&branchName=master) |
```

Comment:
It's late in the game to bring this up, but do we need to specify `pkce` everywhere in the folder structure and project names? I think I would've gone with just `CSVtoOCS` everywhere and left the auth implementation out of it. 
I don't necessarily know if it's worth the effort to change all the paths now, so feel free to ignore this.

Reply:
definately clumsy, will update